### PR TITLE
Point to forked version of python irods client 

### PR DESCRIPTION
**Pull request by kxk302**
that has extra debug statements to troubleshoot Sentry error

### CODE REVIEW

**Comment by kxk302** [comment]
@mvdbeek Could you please review merge this? I want to point Galaxy to a fork of python-irodsclient that has more debugging,  to troubleshoot irods Sentry error. Thanks.

**Comment by mvdbeek** [comment]
You can deploy from your branch, this should not be merged.

**Comment by kxk302** [comment]
You are right!

File: lib/galaxy/config/sample/object_store_conf.xml.sample [hunk 1]
```diff
@@ -49,6 +49,7 @@
             <resource name="demoResc" />
             <zone name="tempZone" />
             <connection host="localhost" port="1247" timeout="30" poolsize="3"/>
```
Comment (kxk302) [inline lib/galaxy/config/sample/object_store_conf.xml.sample:51]: For files larger than 100 MB use iput with 3 threads.

**Pull request closed without merging**